dispatch/static/manager/src/js/util/redux/actions.js

```diff
@@ -1,6 +1,6 @@
 import R from 'ramda'
 import { normalize, arrayOf } from 'normalizr'
-import { push } from 'react-router-redux'
+import { push, replace } from 'react-router-redux'
```

Comment:
by delete,
It makes sense to use replace there too so I'll change it as well, then push won't need to be imported